### PR TITLE
Add Best Cable and Millenium TV to pe.m3u

### PR DESCRIPTION
**Pull request by sguinetti**
This PR adds Best Cable Perú and Millenium TV Pucallpa

### CODE REVIEW

File: channels/pe.m3u [hunk 1]
```diff
@@ -64,3 +64,5 @@ https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="" tvg-logo="https://graph.facebook.com/BestCableTvCanal3/picture?width=320&height=320" group-title="",Best Cable Perú
 #EXTVLCOPT:http-referrer=https://bestcableperu.com.pe/tv3.html
 https://live.siete.us/bestcable/bestcable/index.m3u8
```
Comment (LaneSh4d0w) [inline channels/pe.m3u:66]: This link works for me even without the referer.

File: channels/pe.m3u [hunk 2]
```diff
@@ -64,3 +64,5 @@ https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="" tvg-logo="https://i.imgur.com/zYpMHhQ.png" group-title="Local",Millenium 49 TV (Pucallpa)
+https://stmv.panel.mivideo.pro/mileniuntv/mileniuntv/playlist.m3u8
```
Comment (LaneSh4d0w) [inline channels/pe.m3u:68]: Doesn't work for me -> 404 Not Found.

**Review by BellezaEmporium — CHANGES_REQUESTED** [review]
Please see comments above.

File: channels/pe.m3u (hunk 1 above)
Comment (sguinetti) [reply channels/pe.m3u:65]: Ok. I delete EXTVLCOPT line

File: channels/pe.m3u (hunk 2 above)
Comment (sguinetti) [reply channels/pe.m3u:67]: The channel is https://millenium49tv.com/, because this website opens 8:00-23:00, I tagged [Not 24/7]

Comment (LaneSh4d0w) [reply channels/pe.m3u:67]: Noted.

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**